testing/stable_sort_by_key_large.cu

```diff
@@ -25,8 +25,8 @@ void _TestStableSortByKeyWithLargeKeys(void)
 
     for(size_t i = 0; i < n; i++)
     {
-        h_keys[i] = FixedVector<T,N>(rand());
-        h_vals[i] = i;
+        h_keys[i] = FixedVector<T,N>(static_cast<T>(rand()));
+        h_vals[i] = static_cast<unsigned int>(i);
```

Comment:
There is a random number facility in `unittest` that we should be using here instead.

Reply:
Done, replaced with `unittest::generate_random_integer<T>()(i)`.